onedal/covariance/covariance.cpp

```diff
@@ -56,6 +56,33 @@ struct params2desc {
     }
 };
 
+template <typename Policy, typename Task>
+void init_finalize_compute_ops(pybind11::module_& m) {
+    using namespace dal::covariance;
+    using input_t = partial_compute_result<Task>;
+    m.def("finalize_compute", [](const Policy& policy, const pybind11::dict& params, const input_t& data) {
+        finalize_compute_ops ops(policy, data, params2desc{});
+        return fptype2t{ method2t{ Task{}, ops } }(params);
+    });
+}
+
+template <typename Policy, typename Task>
+void init_partial_compute_ops(pybind11::module_& m) {
```

Comment:
it could be an option to support other constructors from oneDAL, if its necessary

Reply:
Other constructors are not necessary to support current API. It can be added later if it will become important.